src/synopsys-action/service/impl/cloud/github-client-service-cloud.ts

```diff
@@ -0,0 +1,3 @@
+import {GithubClientServiceBase} from '../github-client-service-base'
+
+export class GithubClientServiceCloud extends GithubClientServiceBase {}
```

Comment:
We should proper comment for this class mentioning, why it is left empty

Reply:
Added comments